### DISCUSSION TITLE
How to disable img zoom feature ?

### DISCUSSION THREAD

**Discussion by gVallverdu** [nunocoracao/blowfish#33] [Q&A] [answered]
Hi

Thank you for this nice theme with easy configuration. 

I would like to now if it is possible to disable locally or globally the zoom feature on picture ?

For example, when doing things like that:

    [![FFAAA](/img/test.png)](https://website.com.fr)

The zoom feature make it difficult to use the picture as a link.

Thank you

**Comment by nunocoracao** [comment, marked as answer]
Hello @gVallverdu you can place a nozoom css class in the image(s) where you don't want zoom.

**Reply by gVallverdu** [reply]
Ok
Thank you.

Will this prevent the html code to use the `<figure>` tag ?

If I look at the produced html, I got

```html
    <figure>
      <img
        class="my-0 rounded-md"
        srcset="
          /hugo/img/ffaaa_hudcf3cee0937006603788839194708928_29387_330x0_resize_box_3.png 330w,
          /hugo/img/ffaaa_hudcf3cee0937006603788839194708928_29387_660x0_resize_box_3.png 660w,
          /hugo/img/ffaaa_hudcf3cee0937006603788839194708928_29387_1024x0_resize_box_3.png 1024w,
          /hugo/img/ffaaa_hudcf3cee0937006603788839194708928_29387_1320x0_resize_box_3.png 2x"
        src="/hugo/img/ffaaa_hudcf3cee0937006603788839194708928_29387_660x0_resize_box_3.png"
        alt="FFAAA"
      />
      
    </figure>
```

Is there a way to get a simple `img` tag instead ? It could be nice in some cases, for example to put a picture in a float left div or things like that.

**Reply by nunocoracao** [reply]
You can always use `img` tags in .md files normally. However, your last comment made me think that we should consider enhancing the figure shortcode to make it easier for people to disable zoom.
